msa_sdk/repository.py

```diff
@@ -411,3 +413,44 @@ def get_file(self, file_uri):
         self.path = "{}/file?{}".format(self.api_path, url_encoded)
         self._call_get()
         return json.loads(self.content)
+
+    def add_file(self, file_uri, content):
+        """
+        Add file content.
+
+        Parameters
+        ----------
+            file_uri: String
+            File path in repository
+        content: String
+            File content
+
+        Returns
+        -------
+        None
+
+        """
+        self.action = 'Add file content'
+        url_encoded = urlencode({'uri': file_uri})
+        self.path = "{}/file?{}".format(self.api_path, url_encoded)
```

Comment:
I  think this API should be deprecated, and we should use a real upload API with multipart content. This method may pose serious problem with bianry content.

Reply:
This API is used in WF file editor,  BPM editor ..    since those are from editor, we cannot use mulipart content.   probably we can limit the content size,  sanitize the input path and other security checks

Reply:
Probably I guess we should restrict this kind of API to be only allowed to call only from our UI.  not directly, just thinking out loud

Reply:
What I mean, is we are going to propose a `add_file` api, the idea is to do it correctly. The fact that UI is using this buggy API is another problem.

Reply:
Note:
I am note sure for UI.
Every people on UI should know that you upload things with multi part, so having a 100% in house way to upload things seems to be more a problem than a solution.

I think UI libraries should as default be compatible with multi part upload.

Reply:
@Vignaudo  I created a ticket to address this in the next release - https://jiras.ubiqube.com/browse/MSA-14797